Octokit.Tests.Integration/Clients/UserEmailsClientTests.cs

```diff
@@ -6,15 +6,57 @@ namespace Octokit.Tests.Integration.Clients
 {
     public class UserEmailsClientTests
     {
+        private readonly IUserEmailsClient _emailClient;
+
+        public UserEmailsClientTests()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            _emailClient = github.User.Email;
+        }
+
         [IntegrationTest]
         public async Task CanGetEmail()
         {
-            var github = Helper.GetAuthenticatedClient();
+            var emails = await _emailClient.GetAll();
+            Assert.NotEmpty(emails);
+        }
 
-            var emails = await github.User.Email.GetAll();
+        [IntegrationTest]
+        public async Task CanGetEmailWithApiOptions()
+        {
+            var emails = await _emailClient.GetAll(ApiOptions.None);
             Assert.NotEmpty(emails);
         }
 
+        [IntegrationTest]
+        public async Task ReturnsCorrectCountOfEmailsWithoutStart()
+        {
+            var options = new ApiOptions
+            {
+                PageSize = 5,
+                PageCount = 1
+            };
+
+            var emails = await _emailClient.GetAll(options);
+
+            Assert.Equal(1, emails.Count);
```

Comment:
This test will likely break if the user's account has more than one account - what about if we just did a `Assert.NotEmpty(emails)` here too?